azurerm/internal/services/network/client/client.go

```diff
@@ -1,7 +1,6 @@
 package client
 
 import (
-	networkLegacy "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-03-01/network"
```

Comment:
this is now used for LB's: https://github.com/terraform-providers/terraform-provider-azurerm/blob/8c52648735db0a8b1506178a93687daff75de4fe/azurerm/internal/services/network/client/client.go#L21 - so I think this needs to be left-in & the files re-vendored?

Reply:
Nice catch!